tools/code-owners-parser/ci.yml

```diff
@@ -20,17 +20,10 @@ pr:
   paths:
     include:
       - tools/code-owners-parser
-      - eng/common/scripts/get-codeowners.ps1
+      - eng/common/scripts/get-codeowners
 
 extends:
-  template: /eng/pipelines/templates/stages/archetype-sdk-tool-dotnet.yml
+  template: /eng/common/pipelines/templates/stages/archetype-sdk-tool-pwsh.yml
```

Comment:
Why did your remove the building and production of the .NET tool with this change?

Reply:
My bad, thanks for catching this!